actions/websocket_actions.jsx

```diff
@@ -127,10 +127,14 @@ const pluginEventHandlers = {};
 
 export function initialize() {
     if (!window.WebSocket) {
-        console.log('Browser does not support websocket'); //eslint-disable-line no-console
+        // eslint-disable-next-line no-console
+        console.log('Browser does not support WebSocket');
```

Comment:
In the mobile repo, to avoid so many `eslint-disable-next-line` we changed all console log with proper `LogInfo`, `LogError` or `LogWarning` for logs that actually have to make it into the code. Then, we can still use console.log for testing if we want to, and the linter will only complain where it should.

Sure, not for this PR, but could be an interesting refactor idea to keep the code cleaner in the future.

Reply:
Yeah, that'd be an interesting idea, especially if we want to start logging in the web app more eventually. We used to want to keep the logs clear, but I've started adding more and more to help debugging. We could even start by just removing that ESLint rule, but I do like the small barrier to entry to encourage people to keep the logs cleaner